File: PRAKTIKUM-PEMROGRAMAN1/MODUL2/Python/PRAK204-2410817210024-MuhammadIbnuSina.py
jari=float(input())
tinggi=float(input())

volume= 22/7 * jari**2 *tinggi
luas=2* 22/7 *jari*(jari+tinggi)
keliling=2* 22/7 *jari

print(f"Volume : {volume:.2f}")
print(f"Luas : {luas:.2f}")
print(f"Keliling : {keliling:.2f}")

jari1, tinggi1 = map(float, input().split())

volume1= 22/7 * jari1**2 *tinggi1
luas1=2* 22/7 *jari1*(jari1+tinggi1)
keliling1=2* 22/7 *jari1

print(f"Volume : {volume1:.2f}")
print(f"Luas : {luas1:.2f}")
print(f"Keliling : {keliling1:.2f}")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL2/C/PRAK204-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>
#define PI 22.0/7.0

int main(){
    float jari,tinggi,volume,luas,keliling;

    scanf("%f",&jari);
    scanf("%f",&tinggi);

    volume=PI*jari*jari*tinggi;
    luas=2*PI*jari*(jari+tinggi);
    keliling=2*PI*jari;

    printf("Volume : %.2f\n",volume);
    printf("Luas : %.2f\n",luas);
    printf("Keliling : %.2f\n",keliling);

    float jari1,tinggi1,volume1,luas1,keliling1;

    scanf("%f",&jari1);
    scanf("%f",&tinggi1);

    volume1=PI*jari1*jari1*tinggi1;
    luas1=2*PI*jari1*(jari1+tinggi1);
    keliling1=2*PI*jari1;

    printf("Volume : %.2f\n",volume1);
    printf("Luas : %.2f\n",luas1);
    printf("Keliling : %.2f\n",keliling1);
    return 0;
}